extra/import_data.py

```diff
@@ -1,197 +1,158 @@
-                        else:  # values
-                            variant = {}
-                            for i in range(len(data_headers)):  # iterate headers
-                                key = data_headers[i].replace(" ", "_").replace(".", "\uff0e").lower()
-
-                                # skip everything that starts with father or mother etc since the key
-                                # name contains patient number of parents
-                                # TODO: implement this properly
-                                if key.startswith("father") or key.startswith("mother") \
-                                        or key.startswith("brother") or key.startswith("sister"):
-                                    continue
-
-                                # only use the name inbetween brackets to store, if last char is a right bracket
-                                if key.endswith(")"):
-                                    index_bracket_left = key.rfind("(")
-                                    if index_bracket_left >= 0:
-                                        key = key[index_bracket_left + 1:-1]
-
-                                val = row[i].value
-                                if key == "labels":
-                                    labels = val.split(";")
-                                    val = {}
-                                    for label in labels:
-                                        labelInfo = label.split(",")
-                                        val[labelInfo[0].replace(" ", "_").replace(".", "\uff0e").lower()] = labelInfo[1]
-                                if val is not None and val != "":
-                                    variant[key] = val
-
-                            # add variant info along with metadata to collection
-                            variant.update(patient)
-                            db.insert_one(variant)
-            bar.update(bar.value)
-        logger.debug('finished uploading: %s' % file)
-        # Moves file to other folder, to make uploading a new file (with same filename) possible on the UMCU server (for replacing)
-        os.replace(file, os.path.dirname(__file__) + "/verwerkt/" + os.path.basename(file))
-    print("## \t\tFinished in {0:.2f} minutes".format((time.time() - start_time) / 60))
-
-
-main()
+    '''Main function, for extracting the data from the Alissa output and send to function to parse to MongoDB'''
+     # connection with MongoDB and the correct database "vus" and collection "lastUpdatedOn"
+    mongoclient = pymongo.MongoClient("mongodb://localhost:27017/")
+    db = mongoclient.vus.lastUpdatedOn
+    
+    # Retrieve the last time MongoDB has updated itself, to filter the latest Alissa analyses only
+    lastUpdatedOn_mongoDB = db.find_one({"version" : 1}, {"lastUpdatedOn":1, "_id":0})
+    #If not exist yet, use '2018-01-01T00:00:00.000+0000' and add to mongo (first time use)
+    if lastUpdatedOn_mongoDB is None:
```

Comment:
I prefer this style:`if not lastUpdatedOn_mongoDB:`.

Reply:
I prefer is None for several reasons.
It can return an empty list and I want to avoid below. with "if not" a boolean is called, and with "is None" it will only check if None. I also want to do it consistently like in 
[row 68](https://github.com/UMCUGenetics/VUSualizer/blob/b01ad29cd76a0af00d851473212fe58839be0ff9/extra/import_data.py#L68)
It is also best practice python>3 to use "is None", compared to python2 programming.
[stackoverflow](https://stackoverflow.com/questions/7816363/if-a-vs-if-a-is-not-none)

>>> x = 1
>>> not x
False
>>> x = [1]
>>> not x
False
>>> x = 0
>>> not x
True
>>> x = [0]         # You don't want to fall in this one.
>>> not x
False